vinhloc30796/creativecontact-app
# feat(artwork): enhance artwork page layout and navigation

## Description

This PR enhances the artwork page layout and navigation, improving user experience and providing more detailed artwork information.

## Key Changes

1. Added a "Back to Event" link in the artwork page header
2. Replaced text-based navigation with icon buttons for previous and next artwork
3. Improved layout of artwork details section with scrollable content
4. Expanded artwork description for "The Starry Night" in seed data
5. Added translations for "Back to Event" in English and Vietnamese

## Breaking Changes

- The layout of the artwork page has been significantly modified, which may affect existing styles or components relying on the previous structure.

## Related Issues

- Addresses WEB-157

## Testing Instructions

1. Navigate to an artwork page
2. Verify the new "Back to Event" link functionality
3. Test the new icon-based navigation for previous and next artwork
4. Check the scrollable artwork details section
5. Confirm the expanded description for "The Starry Night" artwork
6. Verify translations for "Back to Event" in English and Vietnamese

## Additional Notes

- The PR introduces new UI components (Button) and icons (ChevronLeft, ChevronRight) from the design system.
- There's a commented-out section for potential future implementation of navigation buttons within the artwork details section.
- The seed data for "The Starry Night" has been significantly expanded to provide a more comprehensive description.
